socialdistribution/app/templates/posts/view_foreign_post.html

```diff
@@ -5,6 +5,40 @@
 {% load filters %}
 {% autoescape off %}
 
+
+{% block javascript %}
+<script>
+    const sendLike = async () => {
+        const like = {
+            summary : '{{user.displayName}} likes your post',
+            author : '{{post.author}}',
+            type : "Like",
+            object : '{{post.author.url}}/posts/{{post.id}}'
```

Comment:
This is only the case for liking posts, liking comments is different.

Reply:
Specifically comments would be 
'{{post.author.url}}/posts/{{post.id}}/comments/{{comment.id}}'